Libraries/src/Amazon.Lambda.AspNetCoreServer/Internal/InvokeFeatures.cs

```diff
@@ -159,16 +159,16 @@ IEnumerator IEnumerable.GetEnumerator()
 
         IHeaderDictionary IHttpRequestFeature.Headers { get; set; } = new HeaderDictionary();
 
-        Stream IHttpRequestFeature.Body { get; set; } = new MemoryStream();
-
+        Stream IHttpRequestFeature.Body { get; set; } = new MemoryStream();
+
 #endregion
-
+

```

Comment:
nit: line endings issue?

Reply:
Updated all of the line endings for InvokeFeatures.cs in a separate commit.